MCCS guidance update + APIM recommendations (#162)
bcgov/public-cloud-techdocs: docs/shared-services/mccs/mccs.md

2. Submit an **on-premises** [firewall request](https://ssbc-client.gov.bc.ca/services/3rdpartygateway/order.htm) (via iStore) to create a firewall rule that allows traffic from the AWS VPC / Azure VNet, through the 3PG firewall and subsequently through the required Zone firewalls
  - Include the **source** (for example your VPC/VNet) and the **destination** (for example your target on-premises network or endpoint) in the request

!!! question "Two requests for three firewalls?"

>     There are at least **3 firewalls** along the connectivity path between AWS/Azure and on-premises resources: the **cloud firewall**, the **3PG firewall**, and the **zone-specific firewalls**. However, from our experience, the on-premises firewall rules are submitted in the **same request form**.

Reviewer: ```suggestion
    There are at least **3 firewalls** along the connectivity path between AWS/Azure and on-premises resources: the **cloud firewall**, the **3PG firewall** and the **zone-specific firewalls**.  In our experience, you can submit all on-premises firewall rules in the **same request form**.
```

>     When you submit an on-premises firewall request, the firewall team will identify which firewalls need to be updated based on the source and destination provided.

Reviewer: ```suggestion
    When you submit an on-premises firewall request, the firewall team identifies which firewalls need updates based on the source and destination information you provide.
```
